### ISSUE TITLE
Fail to load plugin descriptor from file gradle-wrapper.jar

### ISSUE THREAD

**Issue by jeffschaefer1** [markatk/intellij-logitech-craft#2]
I don't see any instructions for installing this. I tried extracting the repository and then going to settings > plugins in phpstorm. I click Install plugin from disk and navigate through the extracted files to the only one it will let me click on: gradle-wrapper.jar and then I get the error: Fail to load plugin descriptor from file gradle-wrapper.jar
Please add installation instructions and/or let me know what I'm doing wrong. Thanks.

**Comment by jeffschaefer1** [comment]
I found your closed issue and I see that you aren't actively working on this right now. At a minimum, I recommend adding that to the readme. I just bought a craft keyboard BECAUSE I saw that this repository existed. I'm certainly not blaming you or trying to force you to provide me with free help, just making a suggestion that might help others. I think I will end up enjoying this keyboard anyway and I may be able to do some of what this plugin has using a hotkey approach. I wish you the best and I hope someone picks this project back up someday.
